openquake/server/tests/test_public_mode.py

```diff
@@ -230,8 +231,10 @@ def test_abort(self):
 
     def test_err_1(self):
         # the rupture XML file has a syntax error
```

Comment:
Please add a comment about why we are disabling the logs here, for future reference

Reply:
Done